code/game/objects/structures/full_window_frame.dm

```diff
@@ -183,3 +182,6 @@
 
 /obj/structure/window_frame/shuttle/blue
 	color = "#6176a1"
+
+/obj/structure/window_frame/shuttle/coalition
```

Comment:
remove, and use new system of `/obj/effect/map_effect/window_spawner/full/shuttle`, more on this below